.github/workflows/docs.yml

```diff
@@ -43,8 +43,6 @@ jobs:
       - uses: actions/checkout@v4.1.7
         with:
           submodules: 'true'
-      - name: Install dependencies, run unit tests and coverage report
-        run: python3 -m pip install --upgrade pip && python3 -m pip install poetry
```

Comment:
This must be why the step's failing, eh?

Reply:
eh